dev/core_generator/lib/src/field_types/bytes_field_type.dart

```diff
@@ -5,15 +5,15 @@ class BytesFieldType extends FieldType {
       : super(
           'Bytes',
           'CoreBytesType',
-          cppName: 'std::vector<uint8_t>',
-          include: '<vector>',
+          cppName: 'const Span<uint8_t>&',
```

Comment:
I wonder: what about 'Span<const uint8_t>' ?

Reply:
Yeah, this was just an easy way to get the full const and ref. I think if we want it to be smarter I'd need to extend the generator a little further.